grakn/concept/type/attribute_type.py

```diff
@@ -59,11 +63,36 @@ def is_string(self):
     def is_datetime(self):
         return False
 
+    def as_boolean(self):
```

Comment:
We weren't able to get all subtypes of `attribute` with value type (e.g:) `boolean`. Using these `as` methods, we can do so.